core/storage/suggestion/gae_models.py

```diff
@@ -285,6 +287,7 @@ def get_translation_suggestions_in_review_with_exp_id(cls, exp_id):
         return (
             cls.get_all()
             .filter(cls.status == STATUS_IN_REVIEW)
+            .filter(cls.language_code == language_code)
```

Comment:
This didn't result in a change to index.yaml? Do we have an index for this query already?

Reply:
No it did not. Will this be problematic? I see multiple entries in index.yaml for the `GeneralSuggestionModel` though: https://github.com/oppia/oppia/blob/develop/index.yaml

Reply:
It's weird that none of them have target_id -- but that would also have been a problem before, I think. 

As long as you've locally "exercised" this function by calling it through the frontend, and nothing changes in index.yaml, I think it should be OK.

Reply:
Yup, I did hit the function locally using the UI. Interestingly, even the commit that introduced this query initially did not have a corresponding edit to `index.yaml`.

Reply:
OK, as long as it works in practice when you test manually, I guess it's probably fine.